### DISCUSSION TITLE
volume xyz has already attached to node xyz with incompatible parameters

### DISCUSSION THREAD

**Discussion by marcinsiembida** [longhorn/longhorn#6917] [Q&A]
hey guys I need your help, I have a problem with attaching volume I will paste screenshot below

I tried attaching to all of the nodes, but after that i get other error saying that I cannot attach to other node, because it is already attached to node-x and when I want to attach it to worker-3, I get error you can see below

![image](https://github.com/longhorn/longhorn/assets/48014776/9eb1e570-76c3-43db-be67-550d868ae48f)

all of my longhorn pods are working and in the web ui everything seems to be ok, but when I want to create some pods for e.g. postgres I am getting this error

**Comment by DamiaSan** [comment]
I see that you opened an issue, so I close this discussion.
